mysql_scripts/table_creation/FullDB.sql

```diff
@@ -129,32 +134,32 @@ CREATE TABLE PasswordReset (
 	FOREIGN KEY (userId) REFERENCES User(userId)
 );
 
-CREATE TABLE Privilege (
-	privilegeId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+CREATE TABLE Permission (
+	permissionId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
 	name VARCHAR(255) NOT NULL,
 	description VARCHAR(500) NOT NULL,
-	tag VARCHAR(255) NOT NULL,
-	CONSTRAINT uniqueTag UNIQUE INDEX(tag)
+	lookupName VARCHAR(255) NOT NULL,
+	CONSTRAINT uniqueLookupName UNIQUE INDEX(lookupName)
 );
 
-CREATE TABLE UserPrivilege (
-	userPrivilegeId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+CREATE TABLE UserPermission (
+	userPermissionId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
 	createdAt DATETIME NOT NULL DEFAULT NOW(),
 	createdBy INTEGER UNSIGNED NOT NULL,
 	FOREIGN KEY(createdBy) REFERENCES User(userId),
 	userId INTEGER UNSIGNED NOT NULL,
 	FOREIGN KEY(userId) REFERENCES User(userId),
-	privilegeId INTEGER UNSIGNED NOT NULL,
-	FOREIGN KEY(privilegeId) REFERENCES Privilege(privilegeId)
+	permissionId INTEGER UNSIGNED NOT NULL,
+	FOREIGN KEY(permissionId) REFERENCES Permission(permissionId)
 );
 
 CREATE TABLE Ability (
 	abilityId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
 	name VARCHAR(255) NOT NULL,
-	tag VARCHAR(255) NOT NULL,
+	lookupName VARCHAR(255) NOT NULL,
     verificationRequired BOOLEAN NOT NULL,
```

Comment:
Lines 160, 167, 168, and 169 have spaces instead of tabs. Otherwise every thing looks good. I'll fix that really quick and merge it.